constants/defaultData.ts

```diff
@@ -0,0 +1,7 @@
+const defaultUser = {
+  name: "User",
+  profilePic:
+    "https://kodaandcrushmarketing.com/wp-content/uploads/2020/11/ToyFaces_Colored_BG_32-7QFYBYH-768x768.jpg",
```

Comment:
might be useful to have a more generic profile picture, like the one for login page